apitest/src/test/java/bisq/apitest/method/wallet/BsqWalletTest.java

```diff
@@ -52,13 +83,164 @@ public void testGetUnusedBsqAddress() {
         NetworkParameters networkParameters = LegacyAddress.getParametersFromAddress(address.substring(1));
         String addressNetwork = networkParameters.getPaymentProtocolId();
         assertNotEquals(PAYMENT_PROTOCOL_ID_MAINNET, addressNetwork);
-        // TODO Fix bug(?) causing the regtest bsq address network to be evaluated as 'testnet' here.
+        // TODO Fix bug causing the regtest bsq address network to be evaluated as 'testnet' here.
         assertTrue(addressNetwork.equals(PAYMENT_PROTOCOL_ID_TESTNET)
                 || addressNetwork.equals(PAYMENT_PROTOCOL_ID_REGTEST));
     }
 
+    @Test
+    @Order(2)
+    public void testInitialBsqBalances(final TestInfo testInfo) {
+        BsqBalanceInfo alicesBsqBalances = getBsqBalances(alicedaemon);
+        log.info("{} -> Alice's BSQ Initial Balances -> \n{}",
+                testName(testInfo),
+                formatBsqBalanceInfoTbl(alicesBsqBalances));
+        verifyBsqBalances(ALICES_INITIAL_BSQ_BALANCES, alicesBsqBalances);
+
+        BsqBalanceInfo bobsBsqBalances = getBsqBalances(bobdaemon);
+        log.info("{} -> Bob's BSQ Initial Balances -> \n{}",
+                testName(testInfo),
+                formatBsqBalanceInfoTbl(bobsBsqBalances));
+        verifyBsqBalances(BOBS_INITIAL_BSQ_BALANCES, bobsBsqBalances);
+    }
+
+    @Disabled // TODO
+    @Test
+    @Order(3)
+    public void testSendBsqAndCheckBalancesBeforeGeneratingBtcBlock(final TestInfo testInfo) {
+        String bobsBsqAddress = getUnusedBsqAddress(bobdaemon);
+        sendBsq(alicedaemon, bobsBsqAddress, SEND_BSQ_AMOUNT);
+        sleep(2000);
+
+        BsqBalanceInfo alicesBsqBalances = getBsqBalances(alicedaemon);
+        BsqBalanceInfo bobsBsqBalances = waitForNonZeroUnverifiedBalance(bobdaemon);
+
+        log.info("BSQ Balances Before BTC Block Gen...");
+        printBobAndAliceBsqBalances(testInfo,
+                bobsBsqBalances,
+                alicesBsqBalances,
+                alicedaemon);
+
+        verifyBsqBalances(expectedBsqBalanceModel(150000000,
+                2500050,
+                0,
+                0,
+                0,
+                0),
+                bobsBsqBalances);
+
+        verifyBsqBalances(expectedBsqBalanceModel(97499950,
+                97499950,
+                97499950,
+                0,
+                0,
+                0),
+                alicesBsqBalances);
+    }
+
+    @Disabled // TODO
+    @Test
+    @Order(4)
+    public void testBalancesAfterSendingBsqAndGeneratingBtcBlock(final TestInfo testInfo) {
+        // There is a wallet persist delay;  we have to
+        // wait for both wallets to be saved to disk.
```

Comment:
Persistence has had some changes to increase the persist frequency, might not need to wait this long

Reply:
I've noticed significant improvement while testing the api on top of v1.5.0.   I'll be checking what wait times I can reduce in apitest.

Reply:
But I could not reduce that wait time without the test failing.  There may be other wait times that can be reduced, like waiting for app startup & shutdown, but not this one.